src/Grids/Grids.jl

```diff
@@ -168,3 +168,33 @@ end
 function find_closest_gridpoint(pt::CylindricalPoint{T}, grid::CartesianGrid3D{T})::NTuple{3,Int} where {T <: SSDFloat}
     find_closest_gridpoint(CartesianPoint(pt),grid)
 end
+
+multiplicity(g::Grid) = prod(multiplicities(g))
+
+function multiplicities(g::CylindricalGrid{T}) where {T}
+    mr = one(T)
+    mφ = T(2π) / width(g.axes[2].interval)

```

Comment:
Before I get something wrong: `width(g.axes[2].interval)` is never `0` because you always extend it to a `_closed_potential`?

And I guess `mφ` should always be an `Integer`, but you make it of type `T` here as it is multiplied with some `T` float in the end (?)

Reply:
Yes, here it can never be 0.

Yes, that was why I did not turned it into an integer. We could though as this is not performance critical. 
I don't have a strong opinion on this one.

Reply:
Then I would leave it as is! Nice work! :)